force-app/main/default/lwc/accountMap/__tests__/accountMap.test.js

```diff
@@ -0,0 +1,80 @@
+import { createElement } from 'lwc';
+import AccountMap from 'c/accountMap';
+import { getRecord } from 'lightning/uiRecordApi';
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+
+// Realistic data with an accounts address details
+const mockGetRecord = require('./data/getRecord.json');
+
+// Register as a LDS wire adapter. Some tests verify the provisioned values trigger desired behavior.
+const getRecordAdapter = registerLdsTestWireAdapter(getRecord);
+describe('c-account-map', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('displays a lightning-map when the wire adaptor returns records', () => {
+        // Create element
+        const element = createElement('c-account-map', {
+            is: AccountMap
+        });
+        document.body.appendChild(element);
```

Comment:
Before the element gets connected to the DOM you should inject a `recordId` like the component was on a record page, see example [here](https://github.com/trailheadapps/lwc-recipes/blob/fb170173c94c5edc4fba50a8b7e0cac38374b010/force-app/main/default/lwc/recordEditFormDynamicContact/__tests__/recordEditFormDynamicContact.test.js#L21).